src/runner.py

```diff
@@ -536,6 +536,9 @@ def _configure_runner(self) -> None:
                 ["/usr/bin/chown", "-R", "ubuntu:ubuntu", "/home/ubuntu/.docker"]
             )
 
+        # Make python an alias of python3
+        self.instance.execute(["ln", "-s", "/usr/bin/python3", "/usr/bin/python"])
```

Comment:
If `/usr/bin/python` exists this command will fail, is this expected?

Reply:
This is run on start a runner LXD instance. Since the LXD instance is created from a ubuntu image, `/usr/bin/python` should be empty.